### ISSUE TITLE
.info validation spits out deprecation warnings on Big Sur (unescaped braces)

### ISSUE THREAD

**Issue by nieder** [fink/fink#211]
On 11.0 using dpkg1.16 branch:
```
$ fink validate languages/perl5184.info 
Validating package file languages/perl5184.info...
Unescaped left brace in regex is deprecated here (and will be fatal in Perl 5.30), passed through in regex; marked by <-- HERE in m/%{ <-- HERE (PatchFile3|default_script)}/ at /opt/sw3/lib/perl5/Fink/Validation.pm line 1037.
Unescaped left brace in regex is deprecated here (and will be fatal in Perl 5.30), passed through in regex; marked by <-- HERE in m/%{ <-- HERE (PatchFile|default_script)}/ at /opt/sw3/lib/perl5/Fink/Validation.pm line 1037.
Unescaped left brace in regex is deprecated here (and will be fatal in Perl 5.30), passed through in regex; marked by <-- HERE in m/%{ <-- HERE (PatchFile2|default_script)}/ at /opt/sw3/lib/perl5/Fink/Validation.pm line 1037.
Package looks good!
```

**Comment by TheSin-** [comment]
Odd I'll look into it.

**Comment by nieder** [comment]
Note that 11.0 is using /usr/bin/perl5.28(.2)

**Comment by TheSin-** [comment]
This is an issue since Xcode 12, I'm getting on a new install on 10.15 just and FYI so I need to fix it today ;)

**Comment by dmacks** [comment]
Fixed in 0e54fb73e736bf18ee648bbebafa69e80db059f0 ?

**Comment by nieder** [comment]
Yes, but will need cherry picking into dpkg1.16

**Comment by dmacks** [comment]
Also needs 17bbea5e

**Comment by nieder** [comment]
Fixed by 0e54fb7 and 17bbea5
